Lib/unittest/test/testmock/testhelpers.py

```diff
@@ -900,6 +901,54 @@ def __getattr__(self, attribute):
         autospec = create_autospec(proxy)
         self.assertFalse(hasattr(autospec, '__name__'))
 
+    def test_spec_inspect_signature(self):
+
+        def myfunc(x, y):
+            pass
+
+        mock = create_autospec(myfunc)
+        assert inspect.getfullargspec(mock) == inspect.getfullargspec(myfunc)
```

Comment:
Let’s change all these asserts into ‘self.asserEquals’ it will give a better error report when the test fails.

Reply:
Changed assert statement to use `assertEqual`. Thanks.